app/assets/javascripts/admin/views/redirect-upload-to-crop.js

```diff
@@ -0,0 +1,58 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function RedirectUploadToCrop (module) {
+    this.module = module
+  }
+
+  RedirectUploadToCrop.prototype.init = function () {
+    this.setupEventListener()
+  }
+
+  RedirectUploadToCrop.prototype.setupEventListener = function () {
+    this.module.addEventListener('submit', function (event) {
+      var filePicker = this.module.querySelector('#edition_images_image_data_file')
+      var file = filePicker.files[0]
+      if (!file) {
+        return
+      }
+
+      event.preventDefault()
+
+      var reader = new FileReader()
+      reader.readAsDataURL(file)
+      reader.onload = function () {
+        var img = document.createElement('img')
+        img.src = reader.result
+        img.onload = function () {
+          if (img.width > 960 && img.height > 640) {
```

Comment:
I think the logic here needs to be inclusive of the size limits to include files which are the right width/height but the wrong height/width
```suggestion
          if ((img.width > 960 && img.height >= 640) || (img.width >= 960 && img.height > 640)) {
```

Reply:
Good point!